__tests__/app.test.js

```diff
@@ -360,3 +360,25 @@ describe("/api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("/api/comments/:comment_id", () => {
+  test("DELETE 204: should delete comment with given comment_id ", () => {
+    return request(app).delete("/api/comments/1").expect(204);
```

Comment:
We could extend this test to test the response body and make sure that it's empty